### DISCUSSION TITLE
Adding the new integration features to HA

### DISCUSSION THREAD

**Discussion by burmjohn** [blakeblackshear/frigate#363] [General]
Blake,

Just got my google coral today, have it working amazing w/the latest 0.8.0 amd64nvidia docker build. Using the GPU to offload the FFMPEG. Absolutely amazing, thank you for what you have made here.
Is there any documentation on the newest integration setup w/.80 on how you setup the additional functionality with HA? I did setup the custom component you made, and was able to add a few entities just wasn’t sure how you get the image of the last detected object for each camera. I see I can get the last image for each object I am tracking but I don’t see a “entity” for the last object detected. BTW the media browser works perfect, was blown away. Going to look more into notifications and how I can use the image in the MQTT. I’m new to HA and frigate, so any suggestions appreciated. Thanks again! John

**Comment by blakeblackshear** [comment]
It should have automatically created a camera entity for each object type you have defined in your config per camera. By default, it is just person. You should see an entity under the integration called "<camera name> Person". That will be an image of the most recent person detected for the camera.

**Reply by burmjohn** [reply]
Ok I see, I wasnt sure if there was also entity for the last object detected overall for each camera. 

Thanks again
